cmd/down.go

```diff
@@ -38,17 +39,22 @@ func executeDown() error {
 			return nil
 		}
 
-		log.Error(err)
+		log.Info(err)
 		return fmt.Errorf("failed to deactivate your cloud native environment")
 	}
 
-	_, client, _, err := GetKubernetesClient(namespace)
+	_, client, _, k8sContext, err := GetKubernetesClient(namespace)
 	if err != nil {
 		return err
 	}
 
 	d, err := deployments.Get(namespace, deployment, client)
 	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
+			fullname := deployments.GetFullName(namespace, deployment)
+			return fmt.Errorf("deployment %s not found [current context: %s]", fullname, k8sContext)
```

Comment:
@rlamana this is the error wheere we include the context. It's the only one that we do it for now.